doc/user/inputs/settings.rst

```diff
@@ -97,3 +97,39 @@ Detail assembly numbers
 
 Detail all assemblies
     The ``detailAllAssems`` setting makes all assemblies in the problem detail assemblies
+
+.. _kinetics-settings:
+
+Kinetics settings
+-----------------
+In reactor physics analyses it is standard practice to represent reactivity
+in either absolute units (i.e., dk/kk' or pcm) or in dollars or cents. To
+support this functionality, the framework supplies the ``beta`` and
+``decayConstants`` settings to apply the delayed neutron fraction and
+precursor decay constants to the Core parameters during initialization.
+
+These settings come with a few caveats:
+
+    1. The ``beta`` setting supports two different meanings depending on
+       the type that is provided. If a float is given, then this setting
```

Comment:
`float` is maybe a bit to programmery for user input docs. Maybe just "If a single value is provided..."

Reply:
Gottcha